utils/build-script-impl

```diff
@@ -3537,24 +3537,27 @@ for host in "${ALL_HOSTS[@]}"; do
                     echo "--install-destdir is required to install products."
                     exit 1
                 fi
-
-                case ${host} in
-                    linux-*)
+                if [[ "$using_xcodebuild" == "TRUE" ]] ; then
+                    case ${host} in
```

Comment:
I think that it will be easier to read this as:

```sh
if [[ ${host} == macosx-* ]] ; then
  set_lldb_xcodebuild_options
  set_lldb_build_mode
  with_pushd ...
  continue
fi
```

At that point, you can even collapse the conditions into:

```sh
if [[ ${host} == macosx-* && ${using_xcodebuild} == TRUE ]] ; then
```